### PR TITLE
Add etc2comp 20170424

### PR DESCRIPTION
**Pull request by intelligide**
Specify library name and version:  **etc2comp/20170424**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/etc2comp/all/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.0)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
```
Comment (uilianries) [inline recipes/etc2comp/all/CMakeLists.txt:5]:
```suggestion
conan_basic_setup()
```

You are not using CONAN_PKG::xxx

File: recipes/etc2comp/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
```
Comment (uilianries) [inline recipes/etc2comp/all/conanfile.py:36]:
```suggestion
            del self.options.fPIC

       def configure(self):
           if self.options.shared:
               del self.options.fPIC
```

File: recipes/etc2comp/all/test_package/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.11)
```
Comment (uilianries) [inline recipes/etc2comp/all/test_package/CMakeLists.txt:1]:
```suggestion
cmake_minimum_required(VERSION 3.1)
```

CXX_STANDARD requires CMake 3.0

File: recipes/etc2comp/all/test_package/test_package.cpp [hunk 4]
```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+
```
Comment (uilianries) [inline recipes/etc2comp/all/test_package/test_package.cpp:2]:
```suggestion
#include <cstdlib>

```

EXIT_SUCCESS is part of cstdlib. There is no invocation for iostream here.

File: recipes/etc2comp/all/conandata.yml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+sources:
+  "20170424":
```
Comment (SSE4) [inline recipes/etc2comp/all/conandata.yml:2]: is it official tag?

Comment (uilianries) [reply recipes/etc2comp/all/conandata.yml:2]: checking again, there is no official tags/releases.

Comment (jgsogo) [reply recipes/etc2comp/all/conandata.yml:2]: then we should use `cci.<date>` version according to : https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases

Comment (jgsogo) [reply recipes/etc2comp/all/conandata.yml:2]: then we should use cci.<date> version according to : https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases

**Pull request merged**